ui/raidboss/data/06-ew/raid/p12s.ts

```diff
@@ -257,6 +271,80 @@ const triggerSet: TriggerSet<Data> = {
     },
+    {
+      id: 'P12S Paradeigma Counter',
+      type: 'StartsUsing',
+      netRegex: { id: '82ED', source: 'Athena', capture: false },
+      run: (data) => data.paradeigmaCounter++,
+    },
+    {
+      id: 'P12S Paradeigma 1 Clones',
+      type: 'Ability',
+      // 8314 appears to transform the orbs ("Ideas") into clones once in position
+      netRegex: { id: '8314', source: 'Thymou Idea' },
+      condition: (data) => data.paradeigmaCounter === 1,
+      delaySeconds: 0.5, // need a small delay to let position data catch up
+      suppressSeconds: 10, // we only need y-pos of one add, and don't refire on the 2nd set of orbs
+      promise: async (data, matches) => {
+        data.combatantData = [];
+        const id = parseInt(matches.sourceId, 16);
+        data.combatantData = (await callOverlayHandler({
+          call: 'getCombatants',
+          ids: [id],
+        })).combatants;
+      },
+      infoText: (data, _matches, output) => {
+        if (data.combatantData.length === 0)
+          return output.clones!({ dir: output.unknown!() });
+        const y = data.combatantData[0]?.PosY;
+        if (y === undefined)
+          return output.clones!({ dir: output.unknown!() });
+        const cloneSide = y > centerY ? 'south' : 'north';
+        return output.clones!({ dir: output[cloneSide]!() });
+      },
+      outputStrings: {
+        clones: {
+          en: 'Clones ${dir}',
+        },
+        north: Outputs.north,
+        south: Outputs.south,
+        unknown: Outputs.unknown,
+      },
+    },
+    // In Ray 1 (Paradeigma2), two adds always spawn north in pairs with PosX of [85, 105] or [95, 115].
+    // Each cleaves 1/4th of the arena. So given one PosX, we can determine the inside/outside safe lanes.
+
+    // TODO: In Ray 2 (SC IIB), the adds have the same cleave width but spawn at [87, 103] or [97, 113].
+    // So "inside east", e.g., is a bit inaccurate.  Because of mech timing, there also isn't time to cross
+    // the arena.  So realistically, this should be combined with SC IIB triggers to indicate whether
+    //  the player needs to move inside or outside to avoid the cleave that will intersect the 2nd orb.
+    // For now, though, display a reminder to avoid the cleaves.
+    {
+      id: 'P12S Ray of Light',
+      type: 'StartsUsing',
+      netRegex: { id: '82EE', source: 'Anthropos' },
+      delaySeconds: 0.7, // cast time: 4.7s; small delay to avoid collision with Engrave 1 Tower Soak
+      suppressSeconds: 1,
+      alertText: (_data, matches, output) => {
+        const x = Math.round(parseFloat(matches.x));
+        if (x === undefined)
+          return output.avoid!();
+        const [inside, outside] = paradeigma2SafeLanes[x] ?? [];
+        if (inside === undefined || outside === undefined)
+          return output.avoid!(); // will fire during Ray 2 (SC IIB)
+        return output.combined!({ inside: output[inside]!(), outside: output[outside]!() });
```

Comment:
Can you have two separate triggers here for insideEastOutsideWest and insideWestOutsideEast, so that if folks want to overwrite them with different calls (like for markers or some other way of specifying) they can?